tests/bdd/features/file_storage.feature

```diff
@@ -10,13 +10,21 @@ Feature: Journals iteracting with the file system in a way that users can see
         And the journal directory should contain
             2013/07/23.txt
 
+    Scenario: Adding entries to a Folder journal with a custom extension should generate date files
+        Given we use the config "empty_folder_with_extension.yaml"
+        When we run "jrnl 23 July 2013: Testing folder journal."
+        Then we should get no error
```

Comment:
Congratulations, your test uncovered an existing bug! 🎉 I filed it in #1894.

Looks like `empty_folder.yaml` got past this issue by having an "empty" file in it: `tests/data/journals/empty_folder/empty`

Feel free to copy that "empty" file to `tests/data/journals/empty_folder_with_extension/empty` to get this working.

Reply:
I can't replicate this. I'm not sure what might have caused the issue. 

I re-ran the tests without the `empty` file in `test/data/journals/empty_folder` and the tests still passed.

Reply:
Looks like bug #1894 is Windows-only, and this is combined with a limitation of git: since we can only commit files (not directories), that `empty_folder` doesn't exist on the remote repository until we push a file in it.